sdk/cosmos/azure-cosmos/test/test_change_feed.py

```diff
@@ -0,0 +1,265 @@
+# The MIT License (MIT)
+# Copyright (c) Microsoft Corporation. All rights reserved.
+
+import time
+import unittest
+import uuid
+from datetime import datetime, timedelta, timezone
+from time import sleep
+
+import pytest
+from _pytest.outcomes import fail
+
+import azure.cosmos.cosmos_client as cosmos_client
+import azure.cosmos.exceptions as exceptions
+import test_config
+from azure.cosmos import DatabaseProxy
+from azure.cosmos.partition_key import PartitionKey
+
+
+@pytest.fixture(scope="class")
+def setup():
+    if (TestChangeFeed.masterKey == '[YOUR_KEY_HERE]' or
+            TestChangeFeed.host == '[YOUR_ENDPOINT_HERE]'):
+        raise Exception(
+            "You must specify your Azure Cosmos account values for "
+            "'masterKey' and 'host' at the top of this class to run the "
+            "tests.")
+    test_client = cosmos_client.CosmosClient(test_config.TestConfig.host, test_config.TestConfig.masterKey),
+    return {
+        "created_db": test_client[0].get_database_client(TestChangeFeed.TEST_DATABASE_ID)
+    }
+
+@pytest.mark.cosmosEmulator
+@pytest.mark.unittest
+@pytest.mark.usefixtures("setup")
+class TestChangeFeed:
+    """Test to ensure escaping of non-ascii characters from partition key"""
+
+    created_db: DatabaseProxy = None
```

Comment:
Are all these being used ?

Reply:
removed unused ones, also updated in test_change_feed_async.py